portfolio/src/main/webapp/index.html

```diff
@@ -145,7 +145,20 @@ <h2> Comments </h2>
 
```

Comment:
(Seems like I cannot comment on the existing part)
in line 143, your type is "comment", is it a valid type? (I thought you can only use "text")